### ISSUE TITLE
Install-DBAFirstResponderKit Issue?

### ISSUE THREAD

**Issue by Kurama2121** [BrentOzarULTD/SQL-Server-First-Responder-Kit#2547]
Is the intention of the command to install a version of the script best suited for the SQL version? When I run this command it installed a version of the First Responder kit from 2015 on my server which is SQL 2014. If its supposed to grab the latest version, it does not appear to be working.

**Comment by BrentOzar** [comment]
No, it should be the current build of the First Responder Kit, however for support on the tool you're using, please go to the place where you downloaded Install-DBAFirstResponderKit. Thanks.
